integration-tests/src/test/java/tachyon/master/rawtable/RawTableMasterIntegrationTest.java

```diff
@@ -69,4 +70,17 @@ public void tooManyColumnsTest() throws Exception {
     mRawTableMaster.createRawTable(new TachyonURI("/testTable"),
         mMasterConf.getInt(Constants.MAX_COLUMNS) + 1, null);
   }
+
+  @Test
+  public void nullMetadataTest() throws Exception {
+    mException.expect(NullPointerException.class);
```

Comment:
Is `null` value expected as input? If not, perhaps add a precondition check with some exception message?